app/front_page/content.html

```diff
@@ -219,7 +219,7 @@
                ng-click="remove(selected)"
                value='Delete selected'>
         <input type='button'
-               class='btn btn-primary btn-sm'
+               class='btn btn-primary btn-sm btn-success'
```

Comment:
Are you not removing `btn-primary` on purpose? The one you're adding and `btn-primary` seem to be mutually exclusive.